### PR TITLE
Update instruction to run PR validation locally

### PR DESCRIPTION
**Pull request by sbidari**

### CODE REVIEW

File: .Rprofile
Comment (dylanhmorris) [inline .Rprofile:1]: I don't think we want to merge this (or the renv files), do we?

**Review by dylanhmorris — CHANGES_REQUESTED** [review]
Guidance looks good. Not sure we want to add the `renv` files to the repo. Was that an accident or on purpose?

File: .Rprofile
Comment (sbidari) [reply .Rprofile:1]: These are required for local PR validation and other hubs seem to have `renv` files on the repo. Not necessary but maybe helpful?

Comment (dylanhmorris) [reply .Rprofile:1]: I think better not. If anything could clash with a user-set renv configuration

Comment (sbidari) [reply .Rprofile:1]: stop tracking in https://github.com/CDCgov/covid19-forecast-hub/pull/7/commits/b70eae81db6a1658da0a1d9bfde0cb10b0aa4d60

**Review by dylanhmorris — APPROVED** [review]
LGTM

**Pull request merged**